prometheus/metrics.go

```diff
@@ -60,8 +60,8 @@ type Histogram struct {
 	Percentile99 *Metric `json:"percentile99"`
 }
 
-// EnvoyHealth is the number of healthy versus total membership (ie. replicas) inside envoy cluster (ie. service)
-type EnvoyHealth struct {
+// EnvoyServiceHealth is the number of healthy versus total membership (ie. replicas) inside envoy cluster (ie. service)
```

Comment:
The comment perhaps takes some implicit knowledge.
In the EnvoyServiceHealth I see Inbound and Outbound, so I guess that comment could be more accurate something like "is the number of healthy traffic versus total membership (i.e. replicas) inside envoy cluster for inbound and outbound traffic " (or something similar, forgive my wording, but I would comment the Inbound/Outbound reference).

Reply:
will do